inc/interactive/class-content.php

```diff
@@ -309,17 +323,20 @@ public function replaceInteractiveTags( $html ) {
 
 		$html5 = new HtmlParser();
 		$dom = $html5->loadHTML( $html );
+		$title = $this->getTitle( $id );
+		$url = wp_get_shortlink( $id );
 		foreach ( $tags as $tag ) {
 			// Replace
 			$elements = $dom->getElementsByTagName( $tag );
 			$element_number = $elements->length;
 			for ( $i = $elements->length; --$i >= 0; ) {
 				$element = $elements->item( $i );
 				$template = $this->blade->render(
-					"interactive.{$tag}", [
-						'title' => $this->getTitle( $id ),
-						'url' => wp_get_shortlink( $id ),
+					'interactive.media', [
```

Comment:
**Note for when we want to group interactive content for export purposes:** 
It generates extra breaklines.
A method I tested and worked for deleting this:
```php
	/**
	 * Remove breaklines after interactive contents elements rendered.
	 *
	 * @param $dom
	 * @return mixed
	 */
	public function removeBreaklinesBetweenInteractiveContents( $dom ) {
		$nodes = $dom->getElementsByTagName( 'div' );
		for ( $i = 0; $i < $nodes->length; $i++ ) {
			$div_element = $nodes->item( $i );
			if ( strpos( $div_element->getAttribute( 'class' ), 'interactive-content' ) !== false ) {
				$interactive_element = $div_element;
				$next_element = $interactive_element->nextSibling;
				if (
					! is_null( $next_element ) &&
					! is_null( $next_element->nextSibling ) &&
					$next_element->nextSibling->nodeName === 'br'
				) {
					$next_element->nextSibling->parentNode->removeChild( $next_element->nextSibling );

				}
			}
		}
		return $dom;
	}
```

Process the whole content at the beginning would be a better solution.